chain/vm/runtime.go

```diff
@@ -159,7 +161,9 @@ func (rs *Runtime) Store() vmr.Store {
 
 func (rt *Runtime) NewActorAddress() address.Address {
 	var b bytes.Buffer
-	if err := rt.Message().Caller().MarshalCBOR(&b); err != nil { // todo: spec says cbor; why not just bytes?
+	sid, _ := rt.ResolveAddress(rt.origin)
```

Comment:
I actually think we should explicitly use the pubkey addr here, the reason being that its more stable than the ID address. Weird example, but if I send funds to my account, and then create an actor, and both of those operations get reorged, then the resultant address would change. If you base it on the pubkey addr, then the resultant address would remain stable

Reply:
I agree with this, and support a spec change here.